crates/cheatnet/src/forking/cache.rs

```
@@ -94,28 +99,24 @@ impl ForkCache {
         block_number: BlockNumber,
         cache_dir: &str,
     ) -> Result<Self> {
-        let (fork_cache_content, cache_file) = {
-            let cache_file_path = cache_file_path_from_fork_config(url, block_number, cache_dir)?;
-            let mut file = OpenOptions::new()
-                .write(true)
-                .read(true)
-                .create(true)
-                .truncate(false)
-                .open(&cache_file_path)
-                .unwrap();
-
-            let mut cache_file_content = String::new();
-            file.read_to_string(&mut cache_file_content)
-                .expect("Could not read cache file: {path}");
-
-            // File was just created
-            let fork_cache_content = if cache_file_content.is_empty() {
-                ForkCacheContent::new()
-            } else {
-                ForkCacheContent::from_str(cache_file_content.as_str())
-            };
+        let cache_file = cache_file_path_from_fork_config(url, block_number, cache_dir)?;
+        let mut file = OpenOptions::new()
+            .write(true)
+            .read(true)
+            .create(true)
+            .truncate(false)
+            .open(&cache_file)
+            .unwrap();
```

Can we bubble an error here?

---

Fixed